### PR TITLE
Added admin controllers and router. POST /v1/login now repsonds with 200.

### PR DESCRIPTION
**Pull request by dradding**

### CODE REVIEW

File: lib/http/controllers/admin/login_user.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var gateway = require(__dirname+'/../../../../');
```
Comment (cdmcnamara) [inline lib/http/controllers/admin/login_user.js]: This isn't required, you can remove this.

Comment (cdmcnamara) [inline lib/http/controllers/admin/login_user.js]: Can we change the name of this file to just login.js? _user insinuates that it is correlated to a user login which it is not.

File: lib/http/routers/admin_router.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+
+var adminController = require(__dirname+'/../controllers/admin/');
+
+var router = new express.Router();
+
+router.post('/login', adminController.login);
+
+module.exports = router;
```
Comment (cdmcnamara) [inline lib/http/routers/admin_router.js]: Git is weird and likes an extra line at the end of every file, could you add it?

File: lib/http/controllers/admin/login_user.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+module.exports = function(req, res) {
+  res.sendStatus(200);
+};
```
Comment (cdmcnamara) [inline lib/http/controllers/admin/login_user.js]: Git is weird and likes an extra line at the end of every file, could you add it?

File: lib/http/controllers/admin/index.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+function bind(action, filename){
+  module.exports[action] = require(__dirname+'/'+filename);
+}
+
+bind('login', 'login_user.js');
```
Comment (cdmcnamara) [inline lib/http/controllers/admin/index.js]: Git is weird and likes an extra line at the end of every file, could you add it?

File: lib/http/routers/admin_router.js (hunk 2 above)
Comment (dradding) [inline lib/http/routers/admin_router.js]: Spaces added

**Pull request merged**